test/modules/linear_boltzmann_solvers/transport_steady/transport_3d_5_cycles_2_bicgstab.lua

```diff
@@ -0,0 +1,138 @@
+-- 3D Transport test with Vacuum BCs.
+-- SDM: PWLD
+-- Test: Max-value1=6.55387e+00
+--       Max-value2=1.02940e+00
+
+num_procs = 4
+
+-- Check num_procs
+if check_num_procs == nil and number_of_processes ~= num_procs then
+  log.Log(
+    LOG_0ERROR,
+    "Incorrect amount of processors. "
+      .. "Expected "
+      .. tostring(num_procs)
+      .. ". Pass check_num_procs=false to override if possible."
+  )
+  os.exit(false)
+end
+
+-- Setup mesh
+meshgen1 = mesh.MeshGenerator.Create({
+  inputs = {
+    mesh.FromFileMeshGenerator.Create({
+      filename = "../../../assets/mesh/Sphere.case",
+    }),
+  },
+  partitioner = mesh.KBAGraphPartitioner.Create({
+    nx = 2,
+    ny = 2,
+    nz = 1,
+    xcuts = { 0.0 },
+    ycuts = { 0.0 },
+  }),
+})
+grid = meshgen1:Execute()
+
+num_groups = 5
+xs_graphite = xs.LoadFromOpenSn("xs_graphite_pure.xs")
+
+strength = {}
+for g = 1, num_groups do
+  strength[g] = 0.0
+end
+
+mg_src0 = lbs.VolumetricSource.Create({ block_ids = { 1 }, group_strength = strength })
+strength[1] = 1.0
+mg_src1 = lbs.VolumetricSource.Create({ block_ids = { 0 }, group_strength = strength })
+
+-- Setup Physics
+pquad0 = aquad.CreateGLCProductQuadrature3DXYZ(4, 8)
+
+lbs_block = {
+  mesh = grid,
+  num_groups = num_groups,
+  groupsets = {
+    {
+      groups_from_to = { 0, num_groups - 1 },
+      angular_quadrature = pquad0,
+      angle_aggregation_type = "single",
+      angle_aggregation_num_subsets = 1,
+      inner_linear_method = "petsc_bicgstab",
+      l_abs_tol = 1.0e-6,
+      l_max_its = 300,
+      gmres_restart_interval = 100,
+    },
+  },
+  xs_map = {
+    { block_ids = { 0, 1 }, xs = xs_graphite },
+  },
+}
+
+lbs_options = {
+  scattering_order = 0,
+  volumetric_sources = { mg_src0, mg_src1 },
+}
+
+phys1 = lbs.DiscreteOrdinatesSolver.Create(lbs_block)
+phys1:SetOptions(lbs_options)
+
+-- Initialize and Execute Solver
+ss_solver = lbs.SteadyStateSolver.Create({ lbs_solver = phys1 })
+
+ss_solver:Initialize()
+ss_solver:Execute()
+
+-- Get field functions
+fflist = lbs.GetScalarFieldFunctionList(phys1)
+
+-- Slice plot
+--slices = {}
+--for k=1,count do
+--    slices[k] = fieldfunc.FFInterpolationCreate(SLICE)
+--    fieldfunc.SetProperty(slices[k],SLICE_POINT,{x = 0.0, y = 0.0, z = 0.8001})
+--    fieldfunc.SetProperty(slices[k],ADD_FIELDFUNCTION,fflist[k])
+--    --fieldfunc.SetProperty(slices[k],SLICE_TANGENT,{x = 0.393, y = 1.0-0.393, z = 0})
+--    --fieldfunc.SetProperty(slices[k],SLICE_NORMAL,{x = -(1.0-0.393), y = -0.393, z = 0.0})
+--    --fieldfunc.SetProperty(slices[k],SLICE_BINORM,{x = 0.0, y = 0.0, z = 1.0})
```

Comment:
I don't think `SetProperty` is a thing any more...

Maybe even delete the whole commented out block...

Reply:
Fixed.